hostscripts/soc-ci/soc-ci

```diff
@@ -126,6 +136,30 @@ def os_mkcloud_available(args):
         sys.exit('no idea where openstack-mkcloud job %s runs' % args.job_id)
 
 
+def os_mkcloud_artifacts(args):
+    """download artifacts for the given job"""
+    conf = _config_get()
+    server = Jenkins(conf['url'],
+                     username=conf['user'],
+                     password=conf['password'])
+    job = server['openstack-mkcloud']
+
+    build = job.get_build(args.job_id)
+    dest_dir = os.path.join(conf['artifacts_dir'], 'os-mkcloud',
+                            '%s' % args.job_id)
+    if os.path.exists(dest_dir):
+        if not args.force:
+            print('Artifacts already available under %s' % dest_dir)
+            sys.exit(0)
+    else:
+        os.makedirs(dest_dir)
+
+    for filename, artifact in build.get_artifact_dict().items():
+        artifact.save_to_dir(dest_dir)
+    print('Artifacts downloaded to %s' % dest_dir)
```

Comment:
How about adding an option to run `unpack-supportconfig` on them after download?  Like in https://github.com/abel-navarro/cifetch/blob/master/cifetch.py but without reinventing wheels.